src/include/sof/gdb/Makefile.am

```diff
@@ -0,0 +1,20 @@
+noinst_LIBRARIES = libgdb.a
```

Comment:
lib will be empty anyway. I guess content of whole file should be something like  `noinst_HEADERS =gdb.h ringbuffer.h`